contracts/cw-ics721-bridge/src/msg.rs

```diff
@@ -90,6 +90,8 @@ pub struct IbcOutgoingMsg {
     pub channel_id: String,
     /// Timeout for the IBC message.
     pub timeout: IbcTimeout,
+    /// Memo to add custom string to the msg
+    pub memo: Option<String>,
 }
 
 #[cw_serde]
```

Comment:
we need to add a query to this to make the new token metadata accessible. i'd suggest renaming `Metadata { class_id }` to `ClassMetadata { class_id }` and adding an additional `TokenMetadata { class_id, token_id }` query.